keps/sig-scheduling/1923-prefer-nominated-node/README.md

```diff
@@ -163,19 +163,19 @@ _This section must be completed when targeting alpha to a release._
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g., what if some components will restart
-   mid-rollout?
+  No. This is an opt-in feature and a runtime behavior change, it shouldn't impact
+  already running workloads.
 
 * **What specific metrics should inform a rollback?**
+  The existing metric `scheduler_framework_extension_point_duration_seconds{extension_point="Filter"}`.
```

Comment:
We should specify X% increase of this metric may inform a rollback.

Reply:
Done